hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/recovery/ReconOMMetadataManager.java

```diff
@@ -47,4 +50,59 @@ public interface ReconOMMetadataManager extends OMMetadataManager {
    * @return true if OM Tables are initialized, otherwise false.
    */
   boolean isOmTablesInitialized();
+
+  /**
+   * Return a list of volumes owned by a given user; if user is null, returns
```

Comment:
Pls update comment, API should never return all volumes. It should be bounded by default as 1000 max else based on maxKeys.

Reply:
Updated with mentions of start volume and max volumes.